packages/bot-web-ui/src/pages/dashboard/announcements/announcements.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useEffect, useState } from 'react';
+import { Text } from '@deriv/components';
+import { Notifications as Announcement } from '@deriv-com/ui';
+import { StandaloneBullhornRegularIcon } from '@deriv/quill-icons';
+import { load } from '@deriv/bot-skeleton';
+import { save_types } from '@deriv/bot-skeleton/src/constants/save-type';
+import clsx from 'clsx';
+import { localize } from '@deriv/translations';
+import AnnouncementDialog from './announcement-dialog';
+import { ANNOUNCEMENTS } from './config';
+import './announcements.scss';
+import { DBOT_TABS } from 'Constants/bot-contents';
+import { IconAnnounce, MessageAnnounce, TitleAnnounce } from './announcement-components';
+
+type TAnnouncements = {
+    is_mobile?: boolean;
+    handleTabChange: (item: number) => void;
+};
+
+const Announcements = ({ is_mobile, handleTabChange }: TAnnouncements) => {
+    const [isAnnounceDialogOpen, setIsAnnounceDialogOpen] = useState(false);
+    const [isOpenAnnounceList, setIsOpenAnnounceList] = React.useState(false);
+    const [amountAnnounce, setAmountAnnounce] = useState({} as Record<string, boolean>);
+    const buttonRef = React.useRef<HTMLButtonElement>(null);
+    const wrapperRef = React.useRef<HTMLDivElement>(null);
+    const accumulator_announcement = ANNOUNCEMENTS.ACCUMULATOR_ANNOUNCE;
+    const is_active_announce_1 = amountAnnounce?.announce_1;
+
+    const handleClickOutside = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
```suggestion
    const handleClickOutside = (event: MouseEvent) => {
```
to remove this error we need to make it native event type instead of synthetic event type
<img width="1827" alt="Screenshot 2024-08-13 at 1 18 52 PM" src="https://github.com/user-attachments/assets/a9a16d40-87c4-41c9-acb3-904edc727b39">

Reply:
but if we are not going to use this function then we don't need this change